### DISCUSSION TITLE
Statistics Card

### DISCUSSION THREAD

**Discussion by andynash** [piitaya/lovelace-mushroom#1058] [General]
Just wondering if a card like this is likely to be forthcoming?

The HA statistics card is really useful and avoids the need to build template sensors and utility meters, but I'm not aware of a way to display the resulting statistics in a mushroom card.

An example use case is displaying daily, monthly, annual energy consumption from a total_increasing sensor in kWh.

Thanks!

**Comment by piitaya** [comment]
Hello 🙂
I like the idea of this card but I just need to find time for this because I'm busy with the new more info redesign in HA 😅
Also the main issue is that I need to duplicate lot of code, options and editor to add a new card because mushroom architecture isn't modular.

**Reply by andynash** [reply]
Great, thanks for the update, understand you are busy, and I like the new more-info changes! Maybe not the vertical on/off button, but perhaps that will grow on me 😁
